build.roc

```diff
@@ -0,0 +1,115 @@
+app [main] {
+    cli: platform "https://github.com/roc-lang/basic-cli/releases/download/0.12.0/cf_TpThUd4e69C7WzHxCbgsagnDmk3xlb_HmEKXTICw.tar.br",
+    weaver: "https://github.com/smores56/weaver/releases/download/0.2.0/BBDPvzgGrYp-AhIDw0qmwxT0pWZIQP_7KOrUrZfp_xw.tar.br",
+}
+
+import cli.Task exposing [Task]
+import cli.Cmd
+import cli.Stdout
+import cli.Env
+import cli.Arg
+import weaver.Opt
+import weaver.Cli
+
+RocTarget : [
+    MacosArm64,
+    MacosX64,
+    LinuxArm64,
+    LinuxX64,
+    WindowsArm64,
+    WindowsX64,
+]
+
+main =
+
+    cliParser =
+        Cli.weave {
+            release: <- Opt.flag { short: "r", help: "Release build" },
+        }
+        |> Cli.finish {
+            name: "basic-webserver",
+            version: "",
+            authors: ["Luke Boswell <https://github.com/lukewilliamboswell>"],
+            description: "This build script generates the binaries and packages the platform for distribution.",
+        }
+        |> Cli.assertValid
+
+    when Cli.parseOrDisplayMessage cliParser (Arg.list!) is
+        Ok args -> run args
+        Err message -> Task.err (Exit 1 message)
+
+run = \{ release } ->
+
+    printInfoLine! "generating glue for builtins"
+    Cmd.exec "roc" ["glue", "glue.roc", "crates/", "platform/main.roc"]
+    |> Task.mapErr! ErrGeneratingGlue
+
+    printInfoLine! "getting the native target"
+    target = Env.platform |> Task.await! getNativeTarget
```

Comment:
This is a little confusing. Could we write it instead as

```roc
target =
    Env.platform!
    |> getNativeTarget!
```

Reply:
Done